src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -10,4 +10,5 @@
     public static final String MESSAGE_INVALID_INDEX = "The index provided is invalid";
     public static final String MESSAGE_ITEMS_LISTED = "%1$d %2$s listed!";
 
+    public static final String MESSAGE_INVALID_REDIT_ITEM_INDEX = "Invalid index provided for %s item: %s";
```

Comment:
Not sure how to deal with pluralisation tbh. Index vs indices seems to much if-else hahaha

Reply:
I think it does not matter much?
Could write `index (indices) provided for item(s)` .

Reply:
Okie! Perhaps I’ll leave it as it is then